### PR TITLE
Update landing page dependencies

### LINKED ISSUE #3583: Update landing page dependencies
Currently many deps are outdated there.

update all dependencies to latest possible without breaking page. 

**website/landingpage**

### PR DESCRIPTION
**Pull request by ndricimrr**
**Description**

Changes proposed in this pull request:
- Update dependencies
- 
**Related issue(s)**

Resolves #3583

### CODE REVIEW

File: website/landingpage/dev/package.json [hunk 1]
```diff
@@ -17,26 +17,26 @@
     "what-input": "^5.2.12"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.6",
-    "@babel/preset-env": "^7.9.6",
-    "@babel/register": "^7.9.0",
-    "autoprefixer": "^9.8.0",
-    "babel-loader": "^8.1.0",
-    "browser-sync": "^2.28.1",
+    "@babel/core": "7.23.6",
```
Comment (camelCaseChris) [inline website/landingpage/dev/package.json:20]: This is now a fixed version. Is that intentional?

File: website/landingpage/dev/package.json [hunk 2]
```diff
@@ -17,26 +17,26 @@
...
     "gulp-extname": "^0.2.0",
     "gulp-if": "^2.0.0",
     "gulp-load-plugins": "^2.0.8",
-    "gulp-postcss": "^8.0.0",
+    "gulp-postcss": "^9.0.0",
     "gulp-sass": "^5.1.0",
     "gulp-sourcemaps": "^2.6.4",
     "gulp-uglify": "^3.0.1",
-    "is-svg": ">=4.2.2",
+    "is-svg": "5.0.0",
```
Comment (camelCaseChris) [inline website/landingpage/dev/package.json:39]: Same question as above.

File: website/landingpage/dev/package.json (hunk 1 above)
Comment (ndricimrr) [reply website/landingpage/dev/package.json]: not intentional, changed 👍

File: website/landingpage/dev/package.json [hunk 3]
```diff
@@ -67,6 +67,11 @@
     "panini": {
       "marked": "^4.0.10",
       "highlight.js": "^11.0.0"
+    },
+    "browser-sync" : {
+      "localtunnel": {
+        "axios": "1.6.0"
+      }
```
Comment (camelCaseChris) [inline website/landingpage/dev/package.json:74]: What is this part about? Does it fix some issue?

**Review by camelCaseChris — APPROVED** [review]
Looks good to me

**Pull request merged**